### PR TITLE
Cred lib create list

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: ui/admin/app/templates/scopes/scope/credential-stores/credential-store/credential-libraries/index.hbs [hunk 1]
```diff
@@ -53,7 +53,28 @@
...
+        {{else}}
+          {{! can neither list nor create }}
+          {{t
+            'descriptions.neither-list-nor-create'
+            resource=(t 'resources.credential-library.title_plural')
+          }}
+        {{/if}}
+
       </message.description>
       <message.link
```
Comment (DhariniJeeva) [inline ui/admin/app/templates/scopes/scope/credential-stores/credential-store/credential-libraries/index.hbs:79]: I missed a check here

Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/credential-stores/credential-store/credential-libraries/index.hbs]: fixed

File: ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries.js [hunk 2]
```diff
@@ -3,19 +3,28 @@ import { action } from '@ember/object';
...
   // =methods
+  @service can;
 
   /**
    * Loads all credential libraries under the current credential store.
    * @return {Promise{[CredentialLibraryModel]}}
    */
   model() {
-    const { id: credential_store_id } = this.modelFor(
+    const scope = this.modelFor(
```
Comment (randallmorey) [inline ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries.js:17]: This is getting a credential store, not a scope, so I'd just tweak the constant name to match.

File: ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/index.js [hunk 3]
```diff
@@ -1,3 +1,13 @@
 import Route from '@ember/routing/route';
 
-export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesIndexRoute extends Route {}
+export default class ScopesScopeCredentialStoresCredentialStoreCredentialLibrariesIndexRoute extends Route {
+  // =methods
+
+  setupController(controller) {
+    const scope = this.modelFor(
```
Comment (randallmorey) [inline ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/index.js:7]: Again it's a credential store, not a scope, so I'd update the naming here and in the template.

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/index.js (hunk 3 above)
Comment (DhariniJeeva) [reply ui/admin/app/routes/scopes/scope/credential-stores/credential-store/credential-libraries/index.js]: updated1

File: ui/admin/tests/acceptance/credential-library/list-test.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    urls.orgScope = `/scopes/${instances.scopes.org.id}/scopes`;
+    urls.projectScope = `/scopes/${instances.scopes.project.id}`;
+    urls.credentialStores = `${urls.projectScope}/credential-stores`;
+    urls.credentialStore = `${urls.credentialStores}/${instances.credentialStore.id}`;
+    urls.credentialLibraries = `${urls.credentialStore}/credential-libraries`;
+    urls.credentialLibrary = `${urls.credentialLibraries}/${instances.credentialLibrary.id}`;
+    authenticateSession({});
+  });
+
+  test('Users can navigate to users with proper authorization', async function (assert) {
```
Comment (randallmorey) [inline ui/admin/tests/acceptance/credential-library/list-test.js:62]: navigate to credential libraries*

**Review by randallmorey — CHANGES_REQUESTED** [review]

**Review by randallmorey — APPROVED** [review]

**Pull request merged**